src/clj_kondo/impl/analyzer/namespace.clj

```diff
@@ -414,6 +414,21 @@
                                     :syntax
                                     "namespace name expected"))))
                  'user)
+        _ (let [expected-file-path (-> (str (munge ns-name))
+                                       (str/replace #"\." "/"))
```

Comment:
This probably causes the Windows tests to fail. Insteaf of hardcoding `"/"` use `babashka.fs/file-separator`

Reply:
I think it would be better to do this even:

- Always replace in the filepath "/" with "."
- On Windows, additionally replace in the filepath `babashka.fs/file-separator` with "."
- Munge ns-name
- Then check if filename ends-with? munged ns-name

Reply:
Ok I think I made all the requested changes.